recipe/meta.yaml

```
@@ -38,15 +83,28 @@ test:
     - skopt.space
   requires:
     - pip
+    - pytest
+  source_files:
+    - skopt/tests
```

```suggestion
    - skopt/tests
    - setup.cfg
```

This is so pytest understands markers fast_test slow_test and slow.
Including this will result in less tests being run, you may not need to skip as many in tests_to_skip  as a result.

https://github.com/scikit-optimize/scikit-optimize/blob/v0.9.0/setup.cfg#L21